installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1665,9 +1665,10 @@ INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,
 INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('GCP_VM_Instance_ServiceAccount_Full_CloudAPI_Access','Deny usage of service accounts with full cloud API access for VM instances','Enforce the principle of least privilege to prevent privilege escalation, ensure that compute engine instances are not configured to use default service account with the cloud api access scope set to "Allow full acess to all cloud Apis"','To ensure the principle of least privilege execute one of the two options:\n1.replace default service account with a secure and compliant service account\n2.change the access scope for default service account',NULL,'version-1',NULL,NULL,'2022-09-30','2022-09-30','https://github.com/PaladinCloud/CE/wiki/GCP-Policy#Deny-usage-of-service-accounts-with-full-cloud-API-access-for-VM-instances');
 INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Deny_usage_of_default_service_accounts_for_instances','Deny usage of default service accounts for instances','ensure that your Google Compute Engine instances are not configured to use the default service account','NULL',NULL,'version-1',NULL,NULL,'2022-09-26','2022-09-26','https://github.com/PaladinCloud/CE/wiki/GCP-Policy#Deny-usage-of-default-service-accounts-for-instances');
+INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Enable_Confidential_Computing_for_Compute_Instance','Enable Confidential Computing for Compute Instance','Ensure that the Confidential Computing security feature is enabled for VM instances in order to add protection to your sensitive data in use by keeping it encrypted in memory and using encryption keys that Google doesn't have access to.','NULL',NULL,'version-1',NULL,NULL,'2022-10-04','2022-10-04','https://github.com/PaladinCloud/CE/wiki/GCP-Policy#Disable-IP-forwarding-for-Compute-Engine-Instances');
```

Comment:
Query syntax is incorrect, Please add an escape char in policy description for '